package.json

```diff
@@ -1,5 +1,6 @@
 {
   "private": true,
+  "type": "module",
```

Comment:
The error in the build is here, you can remove this line if it doesn't affect your code